slack/verify-install.sh

```diff
@@ -0,0 +1,8 @@
+# test evaluates the expression as true/false
+# -d true if file exists and is a directory
+
+#test to see if slack already installed through construct 
+test -d $HOME/Applications/"Slack.app"
+
+#checks to see if slack was installed directly
```

Comment:
good notes, it is considered most best formatting to have a space after comments in shell